src/app/expense/add-edit-expence/add-edit-expence.component.html

```diff
@@ -1 +1,48 @@
-<p>add-edit-expence works!</p>
+<main class="p-3">
+
+    <div class="modal-header">
+
+        <!-- if clicked on add/ edit  -->
+        <h1>Add an Expense</h1>
+        <!-- close button -->
+        <button type="button" class="btn-close" data-bs-dismiss="modal" routerLink="/dashboard/group/expense">.</button>
+    </div>
+    <form>
+        <div class="form-group mt-2">
+            <label for="description">Description:</label>
+            <input type="text" class="form-control" id="description" name="description" required>
+        </div>
+
+        <div class="form-group mt-2">
+            <label for="amount">Amount:</label>
+            <div class="input-group">
+                <div class="input-group-prepend">
+                    <span class="input-group-text">INR</span>
+                </div>
+                <input type="number" class="form-control" id="amount" name="amount" min="0" required>
+            </div>
+        </div>
+
+        <div class="form-group mt-2">
+            <label for="date">Date:</label>
+            <input type="date" class="form-control" id="date" name="date" required>
+        </div>
+
+        <div class="form-group mt-2">
+            <label for="split">Split:</label>
+            <select class="form-control" id="split" name="split">
+          <option value="equal">Split equally</option>
+          <option value="unequal">Split unequally</option>
+          <option value="shares">Split by shares</option>
+        </select>
+        </div>
+
+        <div class="form-group mt-2">
+            <label for="participants">Participants:</label>
+            <input type="text" class="form-control" id="participants" name="participants" required>
+        </div>
+
+        <button type="submit" class="btn btn-primary">Add Expense</button>
+        <button type="button" class="btn btn-secondary">Cancel</button>
+    </form>
+</main>
```

Comment:
Add comments, I should not remind this everytime

Reply:
Resolved.